src/tests/functional/shared_test_classes/include/shared_test_classes/single_op/activation.hpp

```diff
@@ -82,11 +82,15 @@ class ActivationLayerTest : public testing::WithParamInterface<activationParams>
     static std::string getTestCaseName(const testing::TestParamInfo<activationParams> &obj);
 
 protected:
+    //TO DO, to be removed after 125993
+    void generate_inputs(const std::vector<ov::Shape>& targetInputStaticShapes) override;
     void SetUp() override;
 };
 
 class ActivationParamLayerTest : public ActivationLayerTest {
 protected:
+    //TO DO, to be removed after 125993
+    void generate_inputs(const std::vector<ov::Shape>& targetInputStaticShapes) override;
```

Comment:
I strongly recommend not to override `generate_inputs`, if it is possible

Reply:
This is necessary until create_and_fill_tensor is fixed for fp16.

Reply:
Do not see any blockers to fix tensor generation for fp16